### PR TITLE
LPS-115710 Remove snapshot

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/MultiBarChart.es.js [hunk 1]
```diff
@@ -119,12 +105,18 @@ export default ({data, field, height, structure, width}) => {
 				}}
 			>
 				<XAxis
+					axisLine={{stroke: blueDark}}
+					dataKey="name"
+					interval={0}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/MultiBarChart.es.js:110]: Nice job! 👏

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/chart/TooltipContent.es.js [hunk 2]
```diff
@@ -72,11 +72,25 @@ describe('Tooltip', () => {
...
+
+		expect(tooltipBodies[0].innerHTML).toBe(
+			'Col 1: 1 entry <b>(16.6%)</b>'
+		);
+		expect(tooltipBodies[1].innerHTML).toBe(
+			'Col 2: 2 entries <b>(33.3%)</b>'
+		);
+		expect(tooltipBodies[2].innerHTML).toBe(
+			'Col 3: 3 entries <b>(50%)</b>'
+		);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/chart/TooltipContent.es.js:89]: Awesome job, @alinedoleron! 😄 🎉

**Comment by natocesarrego** [comment]
@alinedoleron, please fix conflicts and send straightforward to Brian through `ci:forward`. 😉

Thanks.

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alinedoleron** [comment]
new pr: https://github.com/natocesarrego/liferay-portal/pull/648

**Pull request closed without merging**